src/main/java/seedu/eduke8/exception/Eduke8Exception.java

```diff
@@ -0,0 +1,7 @@
+package seedu.eduke8.exception;
+
+public class Eduke8Exception extends Exception {
```

Comment:
Thanks for adding this but would be good to use a separate branch next time

Reply:
Alright noted, happen to have it here because it is linked to the QuizQuestion class.